docs/app/_components/CommandItems.tsx

```diff
@@ -203,11 +203,16 @@ export const PagesItem = ({
   setPages,
 }: PagesItemProps) => {
   const router = useRouter();
-  const goto = (slug: string) => {
-    router.push(`/${slug}`);
+  const params = useSearchParams();
+
+  const goto = ({ slug, hash }: { slug: string; hash?: string }) => {
```

Comment:
`hash` irritates me. I thought first it's a hashed value. I would call it subpage or anchor. Would be more clear to me.

Reply:
sadly that is its name :D see https://developer.mozilla.org/en-US/docs/Web/API/URL/hash